canonical-web-and-design/buy.ubuntu.com
# Update homepage

## Done

- Added Vanilla
- Removed old CSS
- Updated homepage layout, navigation and footer with Vanilla patterns

## QA
- Visit https://canonical-staging.myshopify.com/
- See that the design of the page uses Vanilla patterns, and the layout works across all screen sizes

Fixes #18
